binding.gyp

```diff
@@ -35,7 +35,7 @@
             "src/watchman/WatchmanBackend.cc",
             "src/shared/BruteForceBackend.cc",
             "src/linux/InotifyBackend.cc",
-            "src/unix/fts.cc"
+            "src/unix/legacy.cc"
```

Comment:
TODO: Figure out a way to use fts when using musl, as alpine includes Large file system support and builds fts just fine :)